### DISCUSSION TITLE
Events

### DISCUSSION THREAD

**Discussion by SPAS79** [orbat-mapper/orbat-mapper#223] [Q&A] [answered]
I have seen this raised as an issue and closed as it was implemented in v 1.0.0, but cannot find any way to add, edit or manage events. The events panel is just blank. 

What am I missing?

**Comment by kjellmf** [comment, marked as answer]
Scenario events are not properly implemented yet. It is planned for version 1.0.0, but it is not finished.

**Reply by SPAS79** [reply]
Very nice! Will test it at the next opportunity and report back!

**Comment by SPAS79** [comment]
Thanks. I thought I was going mad. I guess those can still be written manually in the JSON? Just a matter of understanding the format.

**Reply by kjellmf** [reply]
I have not finalized the data structure yet, but if you take a look at the [falkland82.json](https://github.com/orbat-mapper/orbat-mapper/blob/db64545f520b343daec07c1b90d78160c89ec63f/public/scenarios/falkland82.json#L2435) file you'll see some of examples of events. If you are familiar with Typescript, there is also the type definition https://github.com/orbat-mapper/orbat-mapper/blob/db64545f520b343daec07c1b90d78160c89ec63f/src/types/scenarioModels.ts#L154

**Reply by SPAS79** [reply]
Thank you!

**Comment by kjellmf** [comment]
Besic scenario events are now available.
